lib/read_file.fz

```diff
@@ -0,0 +1,29 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion standard library feature read_file
+#
+#
+# -----------------------------------------------------------------------
+
+# read_file -- shortcut for io.file.read
+#
+# A handy shortcut for io.file.read, returns the content of the file in the path as an array of u8.
+#
+read_file(path string) => io.file.read path # NYI
```

Comment:
I do not think this is necessary, `read_file "a"` saves only 3 characers compared to `io.file.read "a"`, and then it should be possible to inherit from `io.file` such that one could write `read "a"`.